src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -4597,7 +4602,7 @@
 			buildSettings = {
 				CODE_SIGNING_ALLOWED = NO;
 				CODE_SIGNING_REQUIRED = NO;
-				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
No, I'm not sure, Xcode suggested this change for the DEBUG build. Do you know if we definitely do not want this? Otherwise we might have to try it out, if it breaks something in our build process. So far the CI has not been unhappy about it. What do you think?

Reply:
Okay, if it comes from the Xcode suggested changes, lets give it a try :D